internal/storage/v2/clickhouse/tracestore/dbmodel/to_dbmodel.go

```diff
@@ -130,6 +48,68 @@ type AttributesGroup struct {
 	BytesValues  []string
 }
 
+// ToDBModel Converts the OTel pipeline Traces into multi dbmodel.Trace for batch insertion.
+func ToDBModel(td ptrace.Traces) []Trace {
+	var traces []Trace
+	for _, resourceSpan := range td.ResourceSpans().All() {
+		rs := resourceSpan.Resource()
+		resource := Resource{
+			Attributes: attributesToGroup(rs.Attributes()),
+		}
+		for _, scopeSpan := range resourceSpan.ScopeSpans().All() {
+			sc := scopeSpan.Scope()
+			scope := Scope{
+				Name:       sc.Name(),
+				Version:    sc.Version(),
+				Attributes: attributesToGroup(sc.Attributes()),
+			}
+			for _, sp := range scopeSpan.Spans().All() {
+				span := Span{
+					Timestamp:     sp.StartTimestamp().AsTime(),
+					TraceId:       sp.TraceID().String(),
+					SpanId:        sp.TraceID().String(),
+					ParentSpanId:  sp.ParentSpanID().String(),
```

Comment:
Does clickhouse support a sequence of bytes as a data type? When we convert IDs to string we double the size of the data.